MekHQ/src/mekhq/resources/PersonnelTableMouseAdapter.properties

```diff
@@ -21,6 +21,7 @@ changeCallsign.text=Change Callsign...
 changeMDClass.text=Change Manei Domini Class
 changeMDRank.text=Change Manei Domini Rank
 changePortrait.text=Change Portrait...
+randomizePortrait.text=Randomize Portrait...
```

Comment:
Usually, we use "..." to indicate that there's going to be some kind of popup. So, I think we should get rid of the ...